wstakemgr/error.go

```diff
@@ -80,6 +84,7 @@ var errorCodeStrings = map[ErrorCode]string{
 	ErrPoolUserInvalTcktsNotFound: "ErrPoolUserInvalTcktsNotFound",
 	ErrBadPoolUserAddr:            "ErrBadPoolUserAddr",
 	ErrStoreClosed:                "ErrStoreClosed",
+	ErrExtraVoteBitsLen:           "ErrExtraVoteBitsLen",
```

Comment:
probably makes more sense to have a single error for any kind of invalid data saved to the db.  I think this already exists in some of the other packages as ErrData.